src/css/styles.css

```diff
@@ -2,6 +2,124 @@
 @import './global.css';
 
 body {
-  font-family: 'Nunito', sans-serif; /* Using Nunito font family*/
-  background-color: var(--primary-color); /*using a css variable*/
+  background-color: var(--light-blue-color);
+}
+
+.meals {
+  max-width: 774px;
+  margin: auto;
+}
+
+.title {
+  font-family: 'Permanent Marker', cursive;
+  height: 35px;
+  font-size: 24px;
+  color: var (--dark-gray-color);
+  font-weight: 400;
+  line-height: 35px;
+  margin-left: 10px;
+}
+
+.meals-card {
+  display: grid;
+  grid-template: repeat(2, 50%) / repeat(3, 33.3%);
```

Comment:
to handle undefined number of meals cards we could set the rows to auto something like `grid-template: auto / repeat(3, 33.3%);`